src/main/resources/static/js/projects/submissions-controller.js

```diff
@@ -0,0 +1,53 @@
+angular.module('CapstoneSystem')
+    .controller('SubmissionsController', function ($scope, User, Project) {
+        var table = $('#submissionsTable').DataTable( {
+            responsive: true
+        } );
+        $scope.isProfOrCoordinator = false;
+
+        User.getCurrentUser().then(function (user) {
+            if(user) {
+                var isUserAprofOrCoordinator = user.role == User.PROFESSOR || user.role == User.COORDINATOR;
+                $scope.isProfOrCoordinator = $scope.isAuthenticated && isUserAprofOrCoordinator;
+            }
+        });
+
+        $scope.editGrade = function() {
+            //open modal
+            $('#gradesModal').modal('show');
+        }
+
+        $scope.addSubmission = function() {
+            //open modal
+            $('#submitModal').modal('show');
+        }
+
+        // Set the date we're counting down to
+        var countDownDate = new Date("April 5, 2018 12:00:00").getTime();
+
+        // Update the count down every 1 second for report submission
+        var x = setInterval(function() {

```

Comment:
good job on this algorithm. I am pretty sure we can use a library to help us with this. 👍

Reply:
We can definitely use a library, I was like it’s not worth it since it’s only a couple lines of code.